src/App.jsx

```diff
@@ -64,20 +70,26 @@ function App() {
       return { ...currForm, [changedField]: newValue };
     });
   };
-  
+
   return (
     <>
-      <div style={{ alignItems: "center" }}>
+      <div className="main-container">
         <Header />
         <MainSection />
-        <ColorPicker options={options} setters={setters} />
-        <InputForm
+        <InputSection
+          setters={setters}
           handleChange={handleChange}
           formData={formData}
           hexColor={hexColor}
         />
+        {/* <ColorPicker options={options} setters={setters} />
+        <InputForm
+          handleChange={handleChange}
+          formData={formData}
+          hexColor={hexColor}
+        /> */}
         <br />
-        <button className='shadow-lg' onClick={handleClick}>
+        <button className="shadow-lg" onClick={handleClick}>
```

Comment:
I think you should include this button within your <InputSection> component, you will need to pass the prop `handleClick` into it though and receive it as an arguement within the component file as well

Reply:
I agree.  I accidentally left it out as I was hastily getting things pushed out so Nadiia could get started.